### DISCUSSION TITLE
DOIs

### DISCUSSION THREAD

**Discussion by maxkfranz** [PathwayCommons/factoid#1197] [General]
See: https://www.crkn-rcdr.ca/en/about-crkn

**Comment by jvwong** [comment]
Just noting that as of now, Editors at eLife and Nature Springer have explicitly asked about minting DOIs. In practice, this enables them to associate data/records with articles via CrossRef.

**Comment by jvwong** [comment]
# [U of T Dataverse (Borealis)](https://borealisdata.ca/dataverse/toronto)


## Notes from webinar (April 4, 2024)

Host: Jasmine Lefresne (Data Librarian)

* Free space where U of T staff can deposit and share research data
* Considerations
    * Individual file &lt; 5 GB
    * Servers located on site @ U of T 
* Access control
    * Open or restricted sharing
        * Restricted
            * Contact (me) can manually enable access through the system (email)
            * Terms of access text field available
    * Embargo 
        * Set access date
        * Dataset always visible
    * Removal
        * De-accession dataset (lose access) but not “delete”
    * Permissions
        * Users and groups (Borealis/Dataverse account required)
* Indexing
    * Discoverable via Google Datasets, DataCite
* Persistent identifiers
    * Assigned DOI at published dataset level 
    * **QUESTION: All versions are listed under the same DOI**
* Versioning
    * Per-dataset; Must republish
    * Record details associated with each version (e.g. file deleted, added) 
    * Previous versions are available, listed together
    * Restricted access can be applied retroactively
* Admin
    * Contact owner on dataset homepage
* Hierarchical organization 
    * Dataverse Collection → Dataset → File(s), Documentation
    * Dataverse Collection
        * Parent Container - e.g. lab or project
        * Imposes constraints on dataset features
            * Identifier (https:/[borealis.ca/dataverse/](borealis.ca/dataverse/)&lt;id>
            * Alternative URL etc.	
    * Dataset
        * Sections: Files, metadata, terms, versions
        * Fields: Host Dataverse, License; Title, Author, Point of contact, Description, Related publication, keywords/Controlled vocab
        * Metadata
            * Curator/contributor attribution
            * Related materials and datasets	
    * File
        * metadata: tags, provenance
    * **My Question: No limit to number of Files and Datasets that researchers can submit to the service (**Subject to future usage constraints)
* Demo
    * Sandbox, no publication DOI
* Webservices API available

**Comment by jvwong** [comment]
# Mockups of Biofactoid Doc in Repositories

Showing demo versions of Biofactoid records in select repositories, namely those that mint DOIs.

## Biofactoid Document

[LD Motif Recognition by Talin: Structure of the Talin-DLC1 Complex. Structure 24 (2016) ](https://biofactoid.org/document/de1b09bf-0104-4d9e-a862-a3bb91d6aade)

<img src="https://github.com/PathwayCommons/factoid/assets/4706307/083478b6-907d-4d0e-9e18-0194da5b0773" width="500px" />

## NDex

The NDEx Project provides an open-source framework where scientists and organizations can store, share, manipulate, and publish biological network knowledge.

Pratt et al. [NDEx, the Network Data Exchange](https://doi.org/10.1016/j.cels.2015.10.001). Cell Syst. Vol. 1, Issue 4: 302-305 (2015).

![www ndexbio org_viewer_networks_f1e304db-f364-11ee-9621-005056ae23aa(iPad Air)](https://github.com/PathwayCommons/factoid/assets/4706307/0090818e-38e3-4615-bd89-3b0eb8f5b8c5)

## [Borealis](https://borealisdata.ca/)  - The Canadian Dataverse Repository

Borealis, the Canadian Dataverse Repository, is a bilingual, multidisciplinary, secure, Canadian research data repository, supported by academic libraries and research institutions across Canada. Borealis supports open discovery, management, sharing, and preservation of Canadian research data.

### Collection

<img width="890" alt="Screenshot 2024-04-05 at 3 30 44 PM" src="https://github.com/PathwayCommons/factoid/assets/4706307/f0c66c3d-3d62-4c30-9a23-d638e869dd1e">

### Dataset

<img width="869" alt="Screenshot 2024-04-05 at 3 31 06 PM" src="https://github.com/PathwayCommons/factoid/assets/4706307/1a13fdd5-cb1d-4ce0-8f62-3b2044624d7d">

#### Dataset: Metadata

<img width="1142" alt="Screenshot 2024-04-05 at 2 55 12 PM" src="https://github.com/PathwayCommons/factoid/assets/4706307/d79647b4-1b91-4826-a66f-9b5d87f0fb98">
